project.clj

```diff
@@ -61,6 +61,7 @@
                  [puppetlabs/jruby-utils "0.7.0"]
                  [puppetlabs/trapperkeeper]
                  [puppetlabs/trapperkeeper-authorization]
+                 [puppetlabs/trapperkeeper-metrics "0.6.0"]
```

Comment:
I think I had been thinking that it would be bad to bump this is tk-metrics because then pe-tk-metrics would be incompatible... but if we release pe-tk-metrics too and then bump both in clj-parent simultaneously, that will work. I'll do that.